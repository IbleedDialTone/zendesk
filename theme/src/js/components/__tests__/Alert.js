import preact from 'preact';
import render from 'preact-render-to-string';

import Alert from '../Alert';

describe('Alert', () => {
	it('renders correctly', () => {
		const tree = render(<Alert>Alert</Alert>);

		expect(tree).toMatchSnapshot();
	});

	it('renders with leading text', () => {
		const tree = render(<Alert leadingText="Leading Text">Alert</Alert>);

		expect(tree).toMatchSnapshot();
	});
});