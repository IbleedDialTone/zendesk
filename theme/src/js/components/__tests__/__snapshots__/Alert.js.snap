// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders correctly 1`] = `"<div class=\\"alert alert-info\\" role=\\"alert\\"><span class=\\"alert-indicator\\"><svg class=\\"lexicon-icon lexicon-icon-info-circle\\"><use xlink:href=\\"#info-circle\\"></use></svg></span><strong class=\\"lead\\"></strong>Alert</div>"`;

exports[`Alert renders with leading text 1`] = `"<div class=\\"alert alert-info\\" role=\\"alert\\"><span class=\\"alert-indicator\\"><svg class=\\"lexicon-icon lexicon-icon-info-circle\\"><use xlink:href=\\"#info-circle\\"></use></svg></span><strong class=\\"lead\\">Leading Text</strong>Alert</div>"`;
